cluster/images/etcd/migrate/migrator.go

```diff
@@ -159,7 +158,7 @@ func (m *Migrator) rollbackEtcd3MinorVersion(current *EtcdVersionPair, target *E
 	if err != nil {
 		return nil, err
 	}
-	err = exec.Command("mv", m.dataDirectory.path, backupDir).Run()
+	err = os.Rename(m.dataDirectory.path, backupDir)
```

Comment:
mv handles different disks... does os.Rename?

Reply:
Looks like the answer is no, although I haven't tried it. There is a open issues about this: https://github.com/cloudfoundry/bosh-utils/issues/10

Reply:
@liggitt this is running inside a container right? do we need to worry about multiple disks?

Reply:
I'm really unfamiliar with how it is used... is all the data internal only or are primary/backup volumes mounted into the container?

Reply:
@liggitt Data never leaves the container, there are no volumes mounted, the mainfest call is here:
https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/manifests/etcd.manifest#L27

the `migrate-if-needed.sh` is here:
https://github.com/kubernetes/kubernetes/blob/master/cluster/images/etcd/migrate-if-needed.sh

The `DATA_DIRECTORY` stays within the container:
https://cs.k8s.io/?q=DATA_DIRECTORY&i=nope&files=&repos=kubernetes/kubernetes

The backup directory is the same as the original directory path with a suffix:
https://github.com/kubernetes/kubernetes/blob/master/cluster/images/etcd/migrate/data_dir.go#L82

Reply:
Ok, thanks